config/locales/en.yml

```
@@ -217,11 +217,14 @@ en:
     update_info_successfully: Update information successfully retrieved
   agents:
     add_agent: New agent added successfully
+    edit_agent: Edit agent  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)